server/CLI/manager.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python3
+# Copyright 2019 Anton Maksimovich <antonio.maksimovich@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=invalid-name,import-error,redefined-outer-name,too-many-arguments,unused-import
+
+"""Utility for controlling(creating/deleting/printing) the clients instances"""
+
+import asyncio
+import os
+import secrets
+import smtplib
+import ssl
+import string
+import sys
+from argparse import ArgumentParser
+from email.message import EmailMessage
+from asyncpg.exceptions import UniqueViolationError
+from dotenv import load_dotenv
+from huntflow_reloaded.models import DB, User, gino_run
+
+load_dotenv()
+
+
+def send_mail(smtp_server, port, sender_email, sender_password, receiver_email,
+              message):
+    """Sending credentials to user. """
+    context = ssl.create_default_context()
+    with smtplib.SMTP_SSL(smtp_server, port, context=context) as server:
+        server.login(sender_email, sender_password)
+        msg = EmailMessage()
+        msg['Subject'] = "Your credentials"
+        msg['From'] = sender_email
+        msg['To'] = receiver_email
+        msg.set_content(message)
+        server.sendmail(sender_email, receiver_email, msg.as_string())
```

Comment:
Should catch the exceptions. See https://docs.python.org/3/library/smtplib.html#smtplib.SMTP.sendmail.